userspace/libsinsp/sinsp.h

```diff
@@ -1125,7 +1125,11 @@ VISIBILITY_PRIVATE
 
 	void get_procs_cpu_from_driver(uint64_t ts);
 
+	struct scap_platform* get_scap_platform();
+
 	scap_t* m_h;
+	struct scap_platform* m_platform {};
+	char m_platform_lasterr[SCAP_LASTERR_SIZE];
```

Comment:
I am not a great fan of this lasterr buffer here, but it's the easy way out for:
* having a long-lived buffer accessible from sinsp
* not introducing yet another getlasterr style helper

The funny thing is we dutifully set the error message in the (linux) platform code but there aren't really any places apart from the call to scap_platform_init where we actually read it, so all that work is wasted. Out of scope for this PR though, IMO.